test/fetchers.test.js

```diff
@@ -356,7 +356,7 @@ describe('testing error page promise resolver', () => {
 describe('testing path info resolution', () => {
   const tests = [
     {
-      url: '/hä',
+      url: '/hä/',
```

Comment:
Do you have a corresponding test without the trailing slash?

Reply:
yes:

https://github.com/adobe/helix-dispatch/blob/df9e22cc73db8ce4b8d7a82120e19eb38b87ae85/test/fetchers.test.js#L376-L387

Reply:
Then we are good to go.